pkg/operator/ceph/config/store.go

```diff
@@ -227,6 +228,25 @@ func StoredFileVolume() v1.Volume {
 				}}}}
 }
 
+// StoredLogVolume returns a pod volume sourced from the stored log files.
+func StoredLogVolume(ContainerLogDir string) v1.Volume {
+	return v1.Volume{
+		Name: logVolumeName,
+		VolumeSource: v1.VolumeSource{
+			HostPath: &v1.HostPathVolumeSource{Path: ContainerLogDir},
```

Comment:
There is already an established pattern for naming here, and it looks slightly off to me. What is currently called `ContainerLogDir` would be more appropriately called `HostLogDir`. The `ContainerLogDir` should be the log dir *inside* the container, and it seems like this is `/var/log/ceph` in all containers. Even though this is constant for all containers, I think it follows in the spirit of what I intended for the `DataPathMap` struct if we populate `ContainerLogDir` with the value from the constant `LogCephDir` and also populate `HostLogDir` with the location on the host.

While we are on this topic, we cannot guarantee that `/var/log/ceph` on the hosts will be writable. The path on host should be within the dataDirHostPath. I think there are two options. Since these are collected by a scraper, it's probably fine to put all the logs into `dataDirHostPath/logs-all-daemons` or something like this. Unless the daemons can clobber each others' logs. It could be better to have a `logs` subdir for individual daemons, e.g. `dataDirHostPath/osd3/logs` if clobbering is a concern.

Reply:
I'm looking into this now.